File: PHP/paypal.php
<?php
require_once('payment.php');
class Paypal extends Payment{
    public $reference;
    public $branch;

    public function __construct($id, $amount, $reference, $branch){
        parent::__construct($id, $amount);
        $this->reference = $reference;
        $this->branch = $branch;
    }
    public function PrintDataPaymentPaypal(){
        echo "Id: $this->id; Amount: $this->amount; Reference: $this->reference; Branch; $this->branch <br/>";
    }
}

File: Java/Paypal.java
public class Paypal extends Payment {
    String reference;
    String branch;

    public Paypal (Integer id, Integer amount, String reference, String branch){
        super(id, amount);
        this.reference = reference;
        this.branch = branch;
    }
    void printDataPaymentPaypal() {
        System.out.println("Id: " + id + "; Amount: " + amount + "; Reference: " + reference + "; Branch: " + branch);
    }
}